Azure AD authentication with new Microsoft Graph API (#3785)
influxdata/docs-v2: content/chronograf/v1.9/administration/managing-security.md

    Note: If you’ve configured TLS/SSL, modify the `PUBLIC_URL` to ensure you're using HTTPS.

#### Configure Azure Active Directory authentication with new Microsoft Graph API

1. [Create an Azure Active Directory application](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#create-an-azure-active-directory-application).
   Note the following information: `<APPLICATION-ID>`, `<TENANT-ID>`, and `<APPLICATION-KEY>`.
   You'll need these to define your Chronograf environment.

2. Be sure to register a reply URL in your Azure application settings.
   This should match the calling URL from Chronograf.
   Otherwise, you will get an error stating no reply address is registered for the application.
   For example, if Chronograf is configured with a `GENERIC_NAME` value of AzureAD, the reply URL would be `http://localhost:8888/oauth/AzureAD/callback`.

3. Your Azure AD application should have below permissions to read user profile.

![image](https://user-images.githubusercontent.com/55806176/154274246-54ad08dd-2f8d-4f2c-95bc-a8c61dd798c5.png)

> 4. After completing the application provisioning within Azure AD, you can now complete the configuration with Chronograf.
>    Using the metadata from your Azure AD instance, proceed to export the following environment variables:
> 
>     Set the following environment variables in `/etc/default/chronograf`:

Reviewer: ```suggestion
4. To update the Chronograf configuration, use your Azure AD instance metadata
to set the following environment variables in `/etc/default/chronograf`:
```

    ```
    GENERIC_TOKEN_URL=https://login.microsoftonline.com/<<TENANT-ID>>/oauth2/token
    TENANT=<<TENANT-ID>>
    GENERIC_NAME=AzureAD
    GENERIC_API_KEY=userPrincipalName
    GENERIC_SCOPES=openid
    GENERIC_CLIENT_ID=<<APPLICATION-ID>>
    GENERIC_AUTH_URL=https://login.microsoftonline.com/<<TENANT-ID>>/oauth2/authorize?resource=https://graph.microsoft.com
    GENERIC_CLIENT_SECRET=<<APPLICATION-KEY>>
    TOKEN_SECRET=secret
    GENERIC_API_URL=https://graph.microsoft.com/v1.0/me
    PUBLIC_URL=http://localhost:8888
    ```

>     Note: If you’ve configured TLS/SSL, modify the `PUBLIC_URL` to ensure you're using HTTPS.

Reviewer: ```suggestion
{{% note %}}
    **Note:** If you’ve configured TLS/SSL, modify the `PUBLIC_URL` to ensure you're using HTTPS.
{{% /note %}}
```
